src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+/**
+ * Created by nguyenminhquang on 9/19/17.
```

Comment:
As you can see from the other classes, we don't normally include author info in class header comments.